docs/tab-widgets/filebeat.asciidoc

```diff
@@ -7,7 +7,7 @@
 .filebeat.yaml
 ----
 filebeat.inputs:
-- type: log
+- type: filestream
   paths: /path/to/logs.json
   json.keys_under_root: true
```

Comment:
`filestream` doesn't have the same syntax for the json tricks.

it looks like

```
- type: filestream
  paths: /path/to/logs.json
  parsers:
      - ndjson:
          keys_under_root: true
          overwrite_keys: true
```

See https://github.com/cyrille-leclerc/my-shopping-cart/blob/2cde6dda3974143271fe9e2d929ffa59849fa94b/elastic-filebeat/filebeat.yml

Reply:
Thanks, I've added two sections now. I've also added instructions for adding metadata processors. LMKWYT.